src/main/kotlin/ca/rmen/poetassistant/restservice/wotd/WotdController.kt

```diff
@@ -20,16 +20,26 @@
 package ca.rmen.poetassistant.restservice.wotd
 
 import ca.rmen.poetassistant.restservice.wotd.model.WotdModel
-import org.springframework.data.domain.Pageable
-import org.springframework.data.domain.Slice
-import org.springframework.data.domain.SliceImpl
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import javax.validation.constraints.Positive
 
 @RestController
+@Validated
 class WotdController(private val service: WotdService) {
 
     @GetMapping("/wotd")
-    fun getWotd(pageable: Pageable): Slice<WotdModel> =
-        SliceImpl(service.findWotdEntries(offset = pageable.offset, size = pageable.pageSize))
+    fun getWotd(
+        @RequestParam("before", required = false)
```

Comment:
Initially I had the following annotation, to specify "today" as the default value:
```kotlin
@RequestParam("after", defaultValue = "#{T(java.time.LocalDate).now()}")
```

Making the parameter not `required`, and using the elvis operator `?:` when passing the request parameters to our service, allows us to avoid any runtime exceptions due to incorrect code in a string for the annotation.